Splitter/TestHelpers/TestExtensions.swift

```diff
@@ -8,7 +8,7 @@
 
 import XCTest
 
-enum AccesID {
+enum AccessID {
```

Comment:
All the references to this need updating :)

Reply:
Woops. thanks